Formula/glibc.rb

```diff
@@ -132,12 +132,12 @@ def install
       system "make", "install"
       prefix.install_symlink "lib" => "lib64"
     end
+  end
 
+  def post_install
     # Install ld.so symlink.
     ln_sf lib/"ld-linux-x86-64.so.2", HOMEBREW_PREFIX/"lib/ld.so"
```

Comment:
Why not just
```ruby
lib.install_symlink "ld-linux-x86-64.so.2" => "ld.so"
```
in the `install` method?

Reply:
That's a very good question.  This code: https://github.com/Homebrew/brew/blob/bf7fe45e8998e56e6690347a0192c454b8cb203b/Library/Homebrew/extend/os/linux/install.rb#L40-L52 looks like it should take care of making that symlink for us if it was not already made by the formula, but I think the code is being run before `glibc` is installed.  So that would mean that `ld.so` does get created before `glibc` is installed, and will point to the wrong file if using brewed `glibc`.  And consequently `brew link` would fail because of a conflicting file.

@sjackman is `symlink_ld_so` always called before we check if `glibc` needs to be installed?  If so, we should probably change the logic of that section because there is no point in checking if the `ld.so` from brewed `glibc` exists if the code is being run before `glibc` is installed.